### PR TITLE
Bump swarmkit to include task reaper fixes and more metrics.

### PR DESCRIPTION
**Pull request by cyli**
This includes the following behavior-modifying PRs:

- https://github.com/docker/swarmkit/pull/2673 Add metrics for store object types
- https://github.com/docker/swarmkit/pull/2669 Fix task reaper batching
- https://github.com/docker/swarmkit/pull/2675 [manager/orchestrator/reaper] Clean out the task reaper dirty set at the end of tick()
- https://github.com/docker/swarmkit/pull/2664 [manager/dispatcher] Replace call to isRunning() to isRunningLocked() in dispatcher Heartbeat()

full diff:
https://github.com/docker/swarmkit/compare/edd5641391926a50bc5f7040e20b7efc05003c26...cf8ac7f3009e714a83f1adc4cc93ce0c20c4b53f

Signed-off-by: Ying Li <ying.li@docker.com>

### CODE REVIEW

File: vendor/archive/tar/LICENSE [hunk 1]
```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2009 The Go Authors. All rights reserved.
```
Comment (thaJeztah) [inline vendor/archive/tar/LICENSE]: Ah, hm, looks like something still went wrong; that's an issue with the latest version of vndr, which removes everything before vendoring.

The `archive/tar` vendor should not be removed

**Review by thaJeztah — CHANGES_REQUESTED** [review]

**Comment by cyli** [comment]
@thaJeztah Oh ok, weird, the vendor CI seemed to pass.  Will do.

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Review by yongtang — APPROVED** [review]
LGTM one all  💚

**Pull request merged**